README.md

```diff
@@ -266,9 +266,9 @@ gcs.credentials.json={"type":"...", ...}
 
 
 # The set of the fields that are to be output, comma separated.
-# Supported values are: `key`, `value`, `offset`, and `timestamp`.
+# Supported values are: `key`, `value`, `offset`, `timestamp` and `headers`.
```

Comment:
Since we stick to Oxford comma in other places, let's keep it uniform
```suggestion
# Supported values are: `key`, `value`, `offset`, `timestamp`, and `headers`.
```